uber/decorators.py

```diff
@@ -110,6 +110,42 @@ def charge(self, session, payment_id, stripeToken, stripeEmail='ignored', **igno
     return charge
 
 
```

Comment:
This needs to be two decorators because we want the caching to take place after the page has been rendered, but we need to mark pages as being cached in a per-function decorator, which is applied at a lower level.